hack/cve-triage/cve_triage_core.py

```
@@ -0,0 +1,2288 @@
+                print("   (dry-run: no changes made to Jira)")
+            print(f"   Report: {report_file}")
+            print(f"{'='*60}\n")
+            return 0
+
+        # Step 5: govulncheck
+        govulncheck_out = _safe_output_path(cve_out_dir, f"govulncheck-{version_tag}.json")
+        print(f"\n[5] Running govulncheck (symbol-level) on {path_branch}...")
+        print(f"  Output: {govulncheck_out}")
+
+        if os.path.exists(govulncheck_out) and not force:
+            print(f"  ℹ️  Using cached govulncheck output (--force to re-run)")
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)